docs/technical_reference/costing/ion_exchange.rst

```diff
@@ -6,54 +6,172 @@ Costing Method Parameters
+   "Cation exchange resin cost", ":math:`c_{res}`", "``cation_exchange_resin_cost``", "153", ":math:`\text{USD}_{2020}\text{/ft}^{3}`", "Assumes strong acid polystyrenic gel-type. From EPA-WBS cost model."
+   "Regenerant dose per volume of resin", ":math:`D_{regen}`", "``regen_dose``", "300", ":math:`\text{kg/}\text{m}^{3}`", "Mass of regenerant chemical per cubic meter of resin volume"
+   "Ion exchange column cost equation A coeff", ":math:`C_{col,A}`", "``vessel_A_coeff``", "1596.499", ":math:`\text{USD}_{2020}`", "Carbon steel w/ stainless steel internals. From EPA-WBS cost model."
+   "Ion exchange column cost equation B coeff", ":math:`C_{col,b}`", "``vessel_b_coeff``", "0.459496", ":math:`\text{dimensionless}`", "Carbon steel w/ stainless steel internals. From EPA-WBS cost model."
+   "Backwash/rinse tank cost equation A coeff", ":math:`C_{bw,A}`", "``backwash_tank_A_coeff``", "308.9371", ":math:`\text{USD}_{2020}`", "Steel tank. From EPA-WBS cost model."
+   "Backwash/rinse tank cost equation B coeff", ":math:`C_{bw,b}`", "``backwash_tank_b_coeff``", "0.501467", ":math:`\text{dimensionless}`", "Steel tank. From EPA-WBS cost model."
+   "Regeneration solution tank cost equation A coeff", ":math:`C_{regen,A}`", "``regen_tank_A_coeff``", "57.02158", ":math:`\text{USD}_{2020}`", "Stainless steel tank. From EPA-WBS cost model."
+   "Regeneration solution tank cost equation B coeff", ":math:`C_{regen,b}`", "``regen_tank_b_coeff``", "0.729325", ":math:`\text{dimensionless}`", "Stainless steel tank. From EPA-WBS cost model."
+   "Fraction of resin replaced per year", ":math:`f_{res}`", "``annual_resin_replacement_factor``", "0.05", ":math:`1/\text{yr}`", "Estimated 4-5% per year. From EPA-WBS cost model."
+   "Minimum hazardous waste disposal cost", ":math:`f_{haz,min}`", "``hazardous_min_cost``", "3240", ":math:`\text{USD}_{2020}\text{/yr}`", "Minimum cost per hazardous waste shipment. From EPA-WBS cost model."
+   "Unit cost for hazardous waste resin disposal", ":math:`f_{haz,res}`", "``hazardous_resin_disposal``", "347.10", ":math:`\text{USD}_{2020}\text{/ton}`", "From EPA-WBS cost model."
+   "Unit cost for hazardous waste regeneration solution disposal", ":math:`f_{haz,regen}`", "``hazardous_regen_disposal``", "3.64", ":math:`\text{USD}_{2020}\text{/gal}`", "From EPA-WBS cost model."
+   "Number of cycles the regenerant can be reused before disposal", ":math:`f_{recycle}`", "``regen_recycle``", "1", ":math:`\text{dimensionless}`", "Can optionally be set by the user to investigate more efficient regen regimes."
+   "Costing factor to account for total installed cost installation of equipment", ":math:`f_{TIC}`", "``total_installed_cost_factor``", "1.65", ":math:`\text{dimensionless}`", "Costing factor to account for total installed cost of equipment"
+   "Unit cost of NaCl", ":math:`c_{regen}`", "``costing.nacl``", "0.09", ":math:`\text{USD}_{2020}\text{/kg}`", "Assumes solid NaCl. From CatCost v 1.0.4"
```

Comment:
Need to correct the symbols in these couple of lines

Reply:
Also, aren't all of these variables constructed on `costing.variable_name`? I don't think you would need to be explicit for these few.

Reply:
> Also, aren't all of these variables constructed on `costing.variable_name`? I don't think you would need to be explicit for these few.

Actually these are chemical costs so would be on ``m.fs.costing`` whereas the rest of the parameters in this table would be on ``m.fs.costing.ion_exchange``. So, when applying the ``cost_ion_exchange`` method, these are created, but the end up somewhere else but are part of the costing process for IX. I think that is why I included them here and made that distinction when I first created this table.

@MarcusHolly I am wondering if you have a preference for how to handle this. It is a bit nuanced and for a reader who is interested in default chemical costs, they might want to know this info but I am happy to remove them. Quickly looking at the docs for the costing package, there isn't anything about costing chemical flows, but that might be a bit too IDAES-y for WaterTAP docs.